File: templates/matlab/control_mpc_decision_boundary.m
function fig = control_mpc_decision_boundary()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('decision_map', 1621, 'advanced MPC control: decision boundary', 'advanced MPC control', 'decision boundary');
end

File: templates/python/control_mpc_decision_boundary.py
"""control_mpc_decision_boundary: MPC 控制进阶决策边界图（decision-map 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='advanced MPC control: decision boundary'):
    return make_template_figure('decision_map', seed=1621, title=title, domain='advanced MPC control', topic='decision boundary')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
